Update Lecture1_OpenScienceEthos.md

SanliFaez opened the pull request:
most of the content was included in the initiation push of this year. The whole lecture plan requires a review by @erikvansebille

erikvansebille commented:
I can't review this PR< because GitHub only allows me to comment/suggest on the changes that you made in this PR (fixing two broken links). Could you please open a PR with the _full_ lecture as a change, so I can comment on it?

SanliFaez commented:
@erikvansebille I changed the file name in the hope of making it open for review but it seems that git is smarter than me. I have changed some more links. Please feel free to add your suggestions in a separate file and I will take care of checking it carefully.
lesson learned not to push files that need a review directly in main.

erikvansebille commented on Lectures/Lecture1_theOpenScienceEthos.md:
```diff
@@ -10,27 +10,28 @@
+Besides that, the international context and especially fair access to scientific knowledge will be discussed, as well as the opposition to the inacted new research policies in connection with the open science movement and the new system of recognition in academia. 
 
 ## Vision of outcome
 + Course participants are able to engage in an informed discussion about the requirements and achievements of the open science movement
```
Why not call them 'students' throughout? Why have you chosen the (more formal) 'course participants'? And if you prefer that, should we also change to that term in other lectures?

SanliFaez replied:
I sometimes think that the course could be useful beyond the UU, hence the use of participants. I agree that we should be consistent. Will do an overall search and replace before moving it to gitlab.

erikvansebille commented on Lectures/Lecture1_theOpenScienceEthos.md:
```diff
@@ -10,27 +10,28 @@
 ## Vision of outcome
 + Course participants are able to engage in an informed discussion about the requirements and achievements of the open science movement
 + Course participants are aware of the sociological barriers to adopting open science practices and the collective actions necessary to overcome those
 + Course participants can appreciate the logical reasoning behind choosing the selection of topics that are covered in this course.
```
Wow, that might be a bit ambitious? Would that not be a goal for the last lecture, when they can look back?

SanliFaez replied:
do you mean the 3rd outcome? you are right, that is perhaps better suited for the symposium goal

erikvansebille commented on Lectures/Lecture1_theOpenScienceEthos.md:
```diff
@@ -10,27 +10,28 @@
+    - The connection between open science and research integrity
     - Access to knowledge and academic emancipation
     - Community-Discipline-Employer
     - The credibility cycle and "laboratory life"
```
Last year, the students particularly liked the Latour discussion. Are we going to go deeper into the sociology of science this year, and if so where/how?

SanliFaez replied:
yes, I will certainly discuss that in my lecture, and it is also covered in the podcast. I will try to discuss the Credibility cycle here only in the context of need for open science practices

erikvansebille commented on Lectures/Lecture1_theOpenScienceEthos.md:
```diff
@@ -39,35 +40,32 @@ In this lecture we will discuss the advancement of the open science movement in
 ## Evaluation and tutorials
 + During the introductory lecture, students are asked to collect the terms and concepts they have heard about for the first time, in this lecture. During the breakout discussions, students are asked to reflect on why they have not learned about these concepts before and the barriers they see in realization of open science ideals.
```
We don't have a breakout discussion as such, do we? Instead, students move to their workgroup rooms. Make that more explicit here?

SanliFaez replied:
let us discuss this. I am on the fence if this should be for the workgroup rooms or in the lecture hall. I understand that it will be chaotic in the lecture hall. Workgroups could be a better option if we find a way to share back the outcome/summary of each group discussion with the rest of the participants

SanliFaez commented:
I have included the suggested changes in a new branch, and to avoid merge headaches, I will close this pull request without merging.
